src/app/FakeLib/WiXHelper.fs

```diff
@@ -588,10 +923,11 @@ let generateWiXScript fileName =
             <Media Id=\"1\" Cabinet=\"media1.cab\" EmbedCab=\"yes\" />
 
             <Directory Id=\"TARGETDIR\" Name=\"SourceDir\">
-              <Directory Id=\"ProgramFilesFolder\" Name=\"ProgramFiles\">
+              <Directory Id=\"@Product.ProgramFilesFolder@\" Name=\"ProgramFiles\">
```

Comment:
I'm not experienced with WiX, but from other `Directory` entries, shouldn't the placeholder be put into `Name`?